lib/saml/ssoe_health_status.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+require 'sentry_logging'
+
```

Comment:
Need `require 'saml/ssoe_health_status'` here

Reply:
isn't that this file?

Reply:
ah I meant ssoe_settings_service, but if it's the same issue with circular dependencies as `health_status` then it should be good to leave it